easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+
+name = 'icc'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_2016_composer_edition_for_cpp_2016.0.109.tgz']
```

Comment:
@DirkdeDraak: did you rename the tarball yourself after downloading?

Because when I download it, I get a tarball named `parallel_studio_xe_2016_composer_edition_for_cpp.tgz`...

Of course, right now it's unclear what Intel will do with the next release, i.e. whether they'll properly version the tarball or not, or just stick to `parallel_studio_xe_2016_composer_edition_for_cpp.tgz` and expect people not to run in trouble.

Maybe it makes sense to add a checksum, to make very sure the used tarball is the right one?

I get this as MD5:

```
f57a892fb494db3c80f20a88aa3e901f  /apps/gent/source/i/icc/parallel_studio_xe_2016_composer_edition_for_cpp.tgz
```